katdal/test/test_datasources.py

```diff
@@ -54,7 +57,7 @@ def to_dask_array(x, chunks=None):
     return da.from_array(x, chunks)
 
 
-def put_fake_dataset(store, prefix, shape, chunk_overrides=None, array_overrides={}, flags_only=False):
+def put_fake_dataset(store, prefix, shape, chunk_overrides=None, array_overrides=None, flags_only=False):
```

Comment:
I know `None` is generally better than an empty container as a default value, but do you gain anything from using it in this function?

Reply:
Once I was adding similar arguments to a bunch of other functions and chaining them through I didn't want to have to remember which ones should default to `None` and which to `{}`.